src/EnergyPlus/TarcogShading.cc

```diff
@@ -588,6 +588,7 @@ namespace TarcogShading {
 		Real64 const s1_2( pow_2( s1 ) );
 		Real64 const s2_2( pow_2( s2 ) );
 		Real64 const s1_s2_2( pow_2( s1 / s2 ) );
+		Real64 const CosTilt = std::cos( tilt );
```

Comment:
This is another one that I differ from the adopted convention.  I prefer the equals, and there are subtle differences, but not for these specific calls.  

These types of things alone don't always hold up a PR from going in, as our style guide doesn't even match the code, which results in some spots of the code adopting new patterns.